examples/example.c

```diff
@@ -290,6 +290,13 @@ main(int argc, char **argv)
         sentry_options_set_crashpad_wait_for_upload(options, true);
     }
 
+    if (has_arg(argc, argv, "attach-view-hierarchy")) {
+        // assuming the example / test is run directly from the cmake build
+        // directory
+        sentry_options_add_typed_attachment(
+            options, "./CMakeCache.txt", VIEW_HIERARCHY, "application/json");
+    }
+
```

Comment:
We want to test this (similar to how we already test attachments); I think it makes sense to add a test for each backend that checks whether the attachment exists & has the `attachment_type:event.view_hierarchy` and `content_type:application/json`.

Reply:
I adapted the existing tests to also send the view-hierarchy attachment & check for its existence (see commit https://github.com/getsentry/sentry-native/pull/1191/commits/27f938f86ff4185f60b1d0aaccd45fc22b0587c9). These already work for backends Inproc/Breakpad/None, so only Crashpad testing is left.